integreat_cms/matomo_api/apps.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from django.apps import apps, AppConfig
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class MatomoApiConfig(AppConfig):
+    """
+    MatomoApiClient config inheriting the django AppConfig
+    """
+
+    name = "integreat_cms.matomo_api"
+
+    def ready(self):
+        """
+        Checking if API is available
+        """
+        # Only check availability if running a server
+        if apps.get_app_config("cms").test_external_apis:
+            if not settings.MATOMO_API_ENABLED:
+                logger.info("Matomo API is disabled")
+            elif settings.DEBUG:
+                logger.info(
+                    "Matomo API is enabled, but in debug mode. "
+                    "Push notifications are really sent, but only to the test region."
+                )
+            else:
+                logger.info("Matomo API is enabled in production mode.")
```

Comment:
```suggestion
```
I you remove the `MATOMO_API_ENABLED` setting and the availability check, I think we can also remove the `ready()` function and don't need to run custom code on initialization of the app.